frontstage/views/account/account.py

```diff
@@ -190,8 +192,16 @@ def something_else_post(session):
         return redirect(url_for("account_bp.something_else"))
     subject = "My account"
     party_id = session.get_party_id()
+    collection_exercise_id = (
+        flask_session["collection_exercise_id"] if "collection_exercise_id" in flask_session else None
+    )
+    period_ref = flask_session["period_ref"] if "period_ref" in flask_session else None
+    survey_id = flask_session["survey_id"] if "survey_id" in flask_session else None
+    survey_ref = flask_session["survey_ref"] if "survey_ref" in flask_session else "0000"
     logger.info("Form validation successful", party_id=party_id)
-    sent_message = _send_new_message(subject, party_id, category="TECHNICAL")
+    sent_message = _send_new_message(
+        collection_exercise_id, survey_id, subject, party_id, period_ref, survey_ref, category="TECHNICAL"
+    )
```

Comment:
This may produce unexpected/inaccurate events depending on the user journey. The `TECHNICAL` category shouldn't be linked to a survey.